crates/host/src/wasmbus/event.rs

```diff
@@ -36,87 +36,91 @@ fn format_actor_claims(claims: &jwt::Claims<jwt::Actor>) -> serde_json::Value {
     }
 }
 
-pub fn actor_started(
+// TODO(#1092): Remove this event in favor of `actor_scaled`
+pub fn actors_started(
     claims: &jwt::Claims<jwt::Actor>,
     annotations: &BTreeMap<String, String>,
-    instance_id: Uuid,
+    host_id: impl AsRef<str>,
+    count: impl Into<usize>,
     image_ref: impl AsRef<str>,
 ) -> serde_json::Value {
     json!({
         "public_key": claims.subject,
         "image_ref": image_ref.as_ref(),
-        "api_version": "n/a",
-        "instance_id": instance_id,
         "annotations": annotations,
+        "host_id": host_id.as_ref(),
         "claims": format_actor_claims(claims),
+        "count": count.into(),
     })
 }
 
-pub fn actor_start_failed(actor_ref: impl AsRef<str>, error: &anyhow::Error) -> serde_json::Value {
-    json!({
-        "actor_ref": actor_ref.as_ref(),
-        "error": format!("{error:#}"),
-    })
-}
-
-pub fn actor_stopped(
+// TODO(#1092): Remove this event in favor of `actor_scaled`
+pub fn actors_start_failed(
     claims: &jwt::Claims<jwt::Actor>,
     annotations: &BTreeMap<String, String>,
-    instance_id: Uuid,
+    host_id: impl AsRef<str>,
+    image_ref: impl AsRef<str>,
+    error: &anyhow::Error,
 ) -> serde_json::Value {
     json!({
         "public_key": claims.subject,
-        "instance_id": instance_id,
```

Comment:
Are any clients depending on this?

Reply:
The diff here is actually really misleading, I only removed `instance_id` from actor_started and actor_stopped which no longer exist.